optimism.tokenlist.json

```diff
@@ -1415,11 +1423,294 @@
+      }
+    },
+    {
+      "chainId": 69,
+      "address": "0x0D760ee479401Bb4C40BDB7604b329FfF411b3f2",
+      "name": "Dola USD Stablecoin",
+      "symbol": "DOLA",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/DOLA.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 1,
+      "address": "0xbb9bc244d798123fde783fcc1c72d3bb8c189413",
+      "name": "TheDAO",
+      "symbol": "TheDAO",
+      "decimals": 16,
+      "logoURI": "https://ethereum-optimism.github.io/logos/theDAO.svg",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0xd8f365c2c85648f9b89d9f1bf72c0ae4b1c36cfd",
+      "name": "TheDAO",
+      "symbol": "TheDAO",
+      "decimals": 16,
+      "logoURI": "https://ethereum-optimism.github.io/logos/theDAO.svg",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 42,
+      "address": "0x8274ea38fe9bea66f8e7c6f3ef742b85d86aeb5d",
+      "name": "TheDAO",
+      "symbol": "TheDAO",
+      "decimals": 16,
+      "logoURI": "https://ethereum-optimism.github.io/logos/theDAO.svg",
+      "extensions": {
+        "optimismBridgeAddress": "0x22F24361D548e5FaAfb36d1437839f080363982B"
+      }
+    },
+    {
+      "chainId": 69,
+      "address": "0x35597dC6f8fDc81d71b311A9E4e2710ef6accb68",
+      "name": "TheDAO",
+      "symbol": "TheDAO",
+      "decimals": 16,
+      "logoURI": "https://ethereum-optimism.github.io/logos/theDAO.svg",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 1,
+      "address": "0x0000000000085d4780B73119b644AE5ecd22b376",
+      "name": "TrueUSD",
+      "symbol": "TUSD",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/TUSD.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0xcB59a0A753fDB7491d5F3D794316F1adE197B21E",
+      "name": "TrueUSD",
+      "symbol": "TUSD",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/TUSD.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 42,
+      "address": "0x1794230aBe1995fcFe6B14865D4B6928121AB9d9",
+      "name": "TrueUSD",
+      "symbol": "TUSD",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/TUSD.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x22F24361D548e5FaAfb36d1437839f080363982B"
+      }
+    },
+    {
+      "chainId": 69,
```

Comment:
Not sure how you did your PR such that it thinks all of these existing tokens are now new but please fix this. Your PR changes should JUST be adding your token and should not be editing other token listings.